base/server/python/pki/server/deployment/__init__.py

```diff
@@ -2649,8 +2649,10 @@ def setup_database_user(self, instance, subsystem):
             'pkidbuser',
             full_name='pkidbuser',
             user_type='agentType',
-            state='1')
-
+            state='1',
+            attributes={
+                "nsPagedSizeLimit": "20000"
```

Comment:
I think Python has a preference for single-quotes, although double-quotes are acceptable too.

Reply:
This is what I saw earlier, but I'm not sure if this is standard or widely accepted:
https://docs.ckan.org/en/ckan-2.1.5/python-coding-standards.html